content/test/unit/ports/contentStorage/MockedStorage.ts

```diff
@@ -1,7 +1,8 @@
+import { ContentItem, IContentStorageComponent } from '@dcl/catalyst-storage'
+import { SimpleContentItem, streamToBuffer } from '@dcl/catalyst-storage/dist/content-item'
 import { Readable } from 'stream'
-import { ContentItem, ContentStorage, SimpleContentItem, streamToBuffer } from '../../../../src/ports/contentStorage/contentStorage'
 
-export class MockedStorage implements ContentStorage {
+export class MockedStorage implements IContentStorageComponent {
```

Comment:
This class can also be removed... There's a `MockedStorage` class in `catalyst-storage`.

Reply:
Needs to be exported first: https://github.com/decentraland/catalyst-storage/pull/53